### PR TITLE
NEXUS-22489 fixed IT updates

### PR DESCRIPTION
**Pull request by sergeyonofreychuck**
ITs now use the actual repository code instead of the version of the plugin from the parent (NXRM)

The issue explains the problem for R format https://issues.sonatype.org/browse/NEXUS-22489

### CODE REVIEW

File: nexus-repository-p2-it/src/test/java/org/sonatype/nexus/repository/p2/P2ITConfig.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package org.sonatype.nexus.repository.p2;
```
Comment (aornatovskyy) [inline nexus-repository-p2-it/src/test/java/org/sonatype/nexus/repository/p2/P2ITConfig.java:1]: missing header

File: nexus-repository-p2-it/pom.xml [hunk 2]
```diff
@@ -77,6 +77,17 @@
       <version>${nxrm-version}</version>
       <type>zip</type>
       <scope>test</scope>
+      <exclusions>
+        <!-- Avoid leaking older r plugins from distro -->
```
Comment (thejosh00) [inline nexus-repository-p2-it/pom.xml:81]: should be p2 instead of r

Comment (sergeyonofreychuck) [reply nexus-repository-p2-it/pom.xml]: https://github.com/sonatype-nexus-community/nexus-repository-p2/pull/68/commits/cf8bef1e36ef43e71039b05d9f422becfd5477b3

**Comment by aornatovskyy** [comment]
This functionality was done in the #64

**Pull request closed without merging**